Tests/DatabaseServiceTests/Repositories/UserRepositoryDeleteTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using DataBaseService.Database;
+using DataBaseService.Database.Models;
+using DataBaseService.Mappers;
+using DataBaseService.Repositories;
+using Kernel.CustomExceptions;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Linq;
+
+namespace DatabaseServiceTests
+{
+    public class UserRepositoryDeleteTests
+    {
+        UserRepository userRepository;
+        DbUserCredential dbUserCredential;
```

Comment:
Good practice to set access modifier explicitly

Reply:
Agree
fixed